src/app/main/component/user/components/profile/edit-profile/edit-profile-form-builder.ts

```diff
@@ -17,7 +20,61 @@ export class EditProfileFormBuilder {
       showLocation: [false],
       showEcoPlace: [false],
       showShoppingList: [false],
-      socialNetworks: ['']
+      socialNetworks: [''],
+      emailPreferences: this.createEmailPreferencesGroup(null)
+    });
+  }
+
+  private createEmailPreferencesGroup(preferences: any[] | null): FormGroup {
```

Comment:
preferences has type  NotificationPreference[]
any  type unsafety and makes the code hard to understand and maintain